web/react/components/sidebar.jsx

```diff
@@ -242,17 +245,17 @@ module.exports = React.createClass({
         var channel = ChannelStore.getCurrent();
         if (channel) {
             if (channel.type === 'D') {
-                var teammate_username = utils.getDirectTeammate(channel.id).username;
-                document.title = teammate_username + ' ' + document.title.substring(document.title.lastIndexOf('-'));
+                var teammateUsername = utils.getDirectTeammate(channel.id).username;
+                document.title = teammateUsername + ' ' + document.title.substring(document.title.lastIndexOf('-'));
```

Comment:
I think we are deliberately not using nicknames/fullnames for PM channels currently, as per #322